Sources/Subs.php

```diff
@@ -1860,75 +1860,55 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 			),
 			array(
 				'tag' => 'code',
-				'type' => 'unparsed_content',
-				'content' => '<div class="codeheader"><span class="code floatleft">' . $txt['code'] . '</span> <a class="codeoperation smf_select_text">' . $txt['code_select'] . '</a> <a class="codeoperation smf_expand_code hidden" data-shrink-txt="' . $txt['code_shrink'] . '" data-expand-txt="' . $txt['code_expand'] . '">' . $txt['code_expand'] . '</a></div><code class="bbc_code">$1</code>',
-				// @todo Maybe this can be simplified?
-				'validate' => isset($disabled['code']) ? null : function(&$tag, &$data, $disabled) use ($context)
+				'type' => 'unparsed_content',,
```

Comment:
Interesting that older PHP doesn't like the double comma.  I'm sure thats a typoe.

Reply:
```suggestion
				'type' => 'unparsed_content',
```